[2024-10-16T09:57:01.944+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-10-16T09:57:01.961+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-10-16T09:57:01.962+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:57:01.963+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:57:01.964+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:57:01.986+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-08-07 00:00:00+00:00
[2024-10-16T09:57:01.993+0000] {standard_task_runner.py:55} INFO - Started process 9338 to run task
[2024-10-16T09:57:02.000+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-08-07T00:00:00+00:00', '--job-id', '1976', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpd_fjaz37']
[2024-10-16T09:57:02.001+0000] {standard_task_runner.py:83} INFO - Job 1976: Subtask insert_rate
[2024-10-16T09:57:02.106+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:57:02.178+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:57:02.263+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-07T00:00:00+00:00
[2024-10-16T09:57:02.265+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 477.54249402, '2024-08-07')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:57:02.281+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:57:02.622+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:57:02.632+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 477.54249402, '2024-08-07')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:57:02.635+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:57:02.653+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240807T000000, start_date=20241016T095701, end_date=20241016T095702
[2024-10-16T09:57:02.713+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:57:02.749+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:06:59.036+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-10-16T10:06:59.060+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-10-16T10:06:59.061+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:59.062+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:06:59.063+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:59.095+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-08-07 00:00:00+00:00
[2024-10-16T10:06:59.104+0000] {standard_task_runner.py:55} INFO - Started process 11869 to run task
[2024-10-16T10:06:59.113+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-08-07T00:00:00+00:00', '--job-id', '2697', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmphdtnwhj4']
[2024-10-16T10:06:59.115+0000] {standard_task_runner.py:83} INFO - Job 2697: Subtask insert_rate
[2024-10-16T10:06:59.329+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-08-07T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:06:59.420+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:06:59.547+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-07T00:00:00+00:00
[2024-10-16T10:06:59.549+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 477.54249402, '2024-08-07')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:06:59.574+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:07:00.350+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:07:00.390+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 477.54249402, '2024-08-07')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:07:00.395+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:07:00.442+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240807T000000, start_date=20241016T100659, end_date=20241016T100700
[2024-10-16T10:07:00.561+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:07:00.627+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
